cmd/minishift/cmd/start.go

```diff
@@ -171,6 +172,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	//Adding active profile information to all instance config
 	addActiveProfileInformation()
 
+	// TODO: force static IP?
```

Comment:
At the moment the API does not return the other possible IP addresses, so I can not check for this in the `preflightCheckAfterStartinghost`

Reply:
Should we however abstract the `Driver.GetIP()`-call and return the static assigned IP address?

Reply:
https://github.com/minishift/minishift/issues/1457#issuecomment-332811221

Reply:
@gbraad Please remove the comment as it is not required for this PR.